packages/server/src/sdk/app/backups/imports.ts

```diff
@@ -170,7 +170,10 @@ export async function importApp(
   appId: string,
   db: Database,
   template: TemplateType,
-  opts: { importObjStoreContents: boolean } = { importObjStoreContents: true }
+  opts: {
+    importObjStoreContents: boolean
+    updateAttachmentColumns?: boolean
+  } = { importObjStoreContents: true, updateAttachmentColumns: true }
```

Comment:
It would be better to make `updateAttachmentColumns` not optional here so we don't accidentally miss usages of it. Defaulting the params like this works well if the entire `opts` param is undefined, but if you explicitly did something like `importApp(id, db, { importObjStoreContents: true })` then `updateAttachmentColumns` would be `undefined`, whereas the default behaviour before this was was to always we do.

We actually do this in `updateWithExport` today, and therefore the behaviour there would change as we would *not* update attachments any more.

Basically all we need to do is make `updateAttachmentColumns` non-optional, then your IDE will correctly flag any missing usages of it (like `updateWithExport`) and prompt you to explicitly specify it and set it to `true`.

Reply:
good spot, thanks cheeks